backend/core/types/DateFieldType.py

```diff
@@ -0,0 +1,25 @@
+
+from graphene import ResolveInfo, String
+from core.models import LettercraftDate
+from core.types.AbstractDjangoObjectType import AbstractDjangoObjectType
+
+
+class LettercraftDateType(AbstractDjangoObjectType):
+    """
+    Type for models that extend the LettercraftDate model.
+    Should not be queried directly, but should be extended by other types.
+    """
+    display_date = String(required=True)
```

Comment:
A computed `@property` field that Graphene does not know how to resolve, so we need to write our own resolver function.